src/platform-includes/session-replay/enable-session-replay/javascript.mdx

```diff
@@ -0,0 +1,22 @@
+Install the session replay package:
+
+```bash {tabTitle: ESM}
+# Using yarn
+yarn add @sentry/replay
+
+# Using npm
+npm install --save @sentry/replay
+```
+
+```html {tabTitle: CDN}
+<script
+  <!--
+    Note that `bundle.replay.min.js` contains both `@sentry/browser` AND
+    `@sentry/replay`, and should therefore be used in place of
+    `@sentry/browser`'s bundle rather than in addition to it.
+  -->
```

Comment:
This bundle doesn't exist and the bundle that's going to be released is an addon bundle, containing **only** the replay integration. So to install the SDK plus Replay, users have to add the follwing bundles:

* `bundle(.tracing)?(.min)?.js`
* `replay(.min)?.js`

So, the setup is identical to how to use our other [pluggable integrations](https://docs.sentry.io/platforms/javascript/configuration/integrations/plugin/#extraerrordata).

I was going to add instructions to the README today. I can also update this example if you want.

Reply:
Update - Here are the readme docs for the CDN bundle: https://github.com/getsentry/sentry-javascript/blob/master/packages/replay/MIGRATION.md